.travis.yml

```diff
@@ -10,6 +10,8 @@ env:
   - TEST_TYPE=cloud
   - TEST_TYPE=integration TEST_VERBOSITY=minimal
   - TEST_TYPE=unit TEST_VERBOSITY=minimal
+  - TEST_TYPE=integration TEST_DOCKER=true TEST_VERBOSITY=minimal
+  - TEST_TYPE=unit TEST_DOCKER=true TEST_VERBOSITY=minimal
```

Comment:
> Both this PR and the openjdk one appear to add to the overall time it takes to run the tests, so maybe we should just rely on the docker tests for our openjdk testing ?

The docker testing is probably adequate to cover openJDK vs oracleJDK, but remember that the docker testing does not test any cloud or bucket tests.  I would argue that we keep both unless we are running too many tests.

> I'm wondering if it makes sense to run both the integration and unit tests - do you know if there is additional cost to us in terms git-lfs bandwith ?

I think it makes sense.  
- Are you sure that no unit tests use files in git-lfs?
- There is a few minutes of additional time to download the git lfs files.  Building the docker image is still the bottleneck.
- I definitely want to see that both types of tests work in the docker image, so we have to do integration.  Also, I worry that the coverage is lower if we only do integration, so I also want unit.  The docker image should become standard in pipelines, so I want it as vetted as possible and I've already seen it fail on unit tests and not integration tests.